etc/config/fortran.amazon.properties

```diff
@@ -72,3 +72,19 @@ compiler.gfortran82.exe=/opt/compiler-explorer/gcc-8.2.0/bin/gfortran
 compiler.gfortran82.name=x86-64 gfortran 8.2
 compiler.gfortransnapshot.exe=/opt/compiler-explorer/gcc-snapshot/bin/gfortran
 compiler.gfortransnapshot.name=x86-64 gfortran (trunk)
+
+###############################
+# Intel Parallel Studio XE for x86
+group.ifort.compilers=ifort16:ifort17:ifort18:ifort19
```

Comment:
```suggestion
group.ifort.compilers=ifort19
```

Reply:
@mattgodbolt  ifort 19 is a nice start, indeed. @RabsRincon I'll disable the other ones in a minute.

https://github.com/mattgodbolt/compiler-explorer-image/pull/127#issue-225985432 made me have another go at crosstool-ng. What I couldn't get to work with the current release version of the tool seems straightforward with their latest git version. Perhaps I can add a few cross-compilers by the end of the week.